src/oaklib/implementations/ontoportal/ontoportal_implementation_base.py

```diff
@@ -50,22 +49,18 @@ def __post_init__(self):
         if self.focus_ontology is None:
             if self.resource:
                 self.focus_ontology = self.resource.slug
-        if not self.OntoPortalClientClass:
-            raise NotImplementedError("OntoPortalClientClass not specified")
-        api_key = get_apikey_value(self.api_key_name)
-        self.client = self.OntoPortalClientClass(api_key=api_key)
+        if not self.ontoportal_client_class:
+            raise NotImplementedError("ontoportal_client_class not specified")
+        api_key = get_apikey_value(self.ontoportal_client_class.name)
+        self.client = self.ontoportal_client_class(api_key=api_key)
 
     def prefix_map(self) -> PREFIX_MAP:
         # TODO
         return {}
 
-    def _get_response(self, path: str, *args, raise_for_status=True, **kwargs):
+    def _get_response(self, *args, **kwargs):
         check_limit()
```

Comment:
do you know if the rate limit is the same for all instances? we could also push the limit checking upstream

Reply:
It's basically a global rate limiter: https://github.com/INCATools/ontology-access-kit/blob/main/src/oaklib/utilities/rate_limiter.py which seems perhaps too conservative, but there may be some good reasons for it that I'm not aware of.

Reply:
By instances, I mean bioportal, agoportal, etc. I know BioPortal says 15 requests per second on https://www.bioontology.org/wiki/Annotator_Optimizing_and_Troublehooting, but finding this kind of specific documentation for all portals is annoying

Reply:
Ah I see, no I don't know off hand if all ontoportal instances have the same rate limits or if it's configurable.